src/main/java/Duke.java

```diff
@@ -1,10 +1,237 @@
+            dateOrTime = command.substring(dashStart + DASH_INDX);
+        } catch (Exception e) {
+            throw new DukeException(LINE + System.lineSeparator() + "Sir, you haven't given me a valid event date "
+                    + "\u2639" + System.lineSeparator() + LINE);
+        }
+        try {
+            taskDescription = command.substring(EVENT_SIZE, dashStart);
+        } catch (Exception e){
+            throw new DukeException(LINE + System.lineSeparator() + "Sir, I'm afraid that command is invalid." + "\u2639"
+                    + System.lineSeparator() + "Please frame your request in this format: event My B'day /at 23/07/1999"
+                    + System.lineSeparator() + LINE);
+        }
+        Event event = new Event((command.substring(EVENT_SIZE, dashStart)), dateOrTime);
+        tasks.add(event);
+        printListSummary(tasks.get(tasks.indexOf(event)));
+    }
+
+    public static void printWelcome() {
+        String logo = "        _|       _|_|        _|_|_|      _|      _|    _|_|_|        _|_|_|\n"
+                + "        _|     _|    _|      _|    _|    _|      _|      _|        _|\n"
+                + "        _|     _|_|_|_|      _|_|_|      _|      _|      _|          _|_|\n"
+                + " _|     _|     _|    _|      _|    _|      _|  _|        _|              _|\n"
+                + "  _| _|    _|  _|    _|  _|  _|    _|  _|    _|    _|  _|_|_|  _|  _|_|_|\n";
+        System.out.print("Initialising...............\n");
+        System.out.println(LINE + System.lineSeparator() + logo);
+        // Prints enclosed text in italics
+        String byline = "\033[3m----------------------------------Just a rather very intelligent system---------\033[0m\n";
+        System.out.println(byline + "Good Evening Sir! I'm J.A.R.V.I.S");
+        System.out.println("How may I be of assistance to you today? " + "\uD83D\uDE01");
+        System.out.println(LINE + System.lineSeparator());
+    }
+
+    public static void printBye() {
+        System.out.println(LINE + System.lineSeparator() + "Affirmative sir, I'll shut down all operations"
+                + System.lineSeparator() + LINE);
+    }
+
+    public static void printList() {
+        System.out.println(LINE + System.lineSeparator() + "Here are the current tasks in your list:");
+        for (Task task : tasks) {
+            System.out.println(tasks.indexOf(task) + 1 + "." + task.printTask());
+        }
+        System.out.println(LINE);
+    }
+
+    public static void printListSummary(Task task) {
+        System.out.println(LINE + System.lineSeparator() + "Will do sir, I've added: "
+                    + System.lineSeparator() + "  " + task.printTask());
+        if (tasks.size() == 1) {
+            System.out.printf("Now you have %d task in your list.\n", tasks.size());
+        } else {
+            System.out.printf("Now you have %d tasks in your list.\n", tasks.size());
+        }
+        System.out.println(LINE);
+    }
```

Comment:
Consider implementing a UI class to handle all print related commands and some of your printing statements are actually constantly reused as well